react/scripts/watch-changes.js

```diff
@@ -5,7 +5,7 @@ const path = require('path');
 const directoryPath = path.join(__dirname, '..', 'src');
 
 const buildAndMoveFiles = () => {
-  exec('yarn parcel build --no-optimize && yarn move', (error, stdout, stderr) => {
```

Comment:
unfortunately `--no-optimiize` for dev has some problems... i was getting errors, so I just removed it and on dev we will work with a minimized bundle instead 😢